torchbenchmark/models/sage/__init__.py

```diff
@@ -0,0 +1,36 @@
+from torchbenchmark.util.framework.gnn.model_factory import GNNModel
+from torchbenchmark.tasks import COMPUTER_VISION
+import os.path as osp
+from torch_geometric.loader import NeighborLoader
+import torch_geometric.transforms as T
+from torch_geometric.datasets import Reddit
```

Comment:
Is `torch_geometric` needed? If so, we need to add `install.py` and `requirements.txt` files which installs the model dependencies.

Reply:
Have moved to `requirements.txt`.